_templates/release.qmd

```diff
@@ -10,3 +10,18 @@ We are very excited to announce the release of a new {{ pkgname }} version {{ v
 Here is an automatically generated summary of the changes in this version.
 
 {{ notes }}
+
+## Acknowledgements
+
+```{r ctbs, results='asis', echo=FALSE}
+releases <- suppressMessages(gh::gh("GET /repos/{owner}/{repo}/releases", 
+               owner = "epiverse-trace",
+               repo = "{{ pkgname }}"))
+ctbs <- suppressMessages(usethis::use_tidy_thanks(
+  paste0('epiverse-trace/', 
+    "{{ pkgname }}"), 
+  from = releases[[length(releases) - 1]][["tag_name"]], 
```

Comment:
One potential additional complication is that we're skipping patch releases for blog posts.

Ideally, we would take all contributors since the last minor release. It requires making some assumptions about the format of release numbers but maybe that's reasonable within our project?

Reply:
That's true! I completely forgot about this. Going back to adjust.

And indexing is correct - it should be `[[2]]` and the test scenario was a fluke (simulist has 3 releases so I accidentally used the right index by doing `length - 1`).